project_milestone_status/models/project_milestone.py

```
@@ -0,0 +1,32 @@
+from odoo import api, fields, models
+
+
+class ProjectMilestone(models.Model):
+    _inherit = "project.milestone"
+
+    execution = fields.Integer(compute="_compute_execution")
+    dedication = fields.Integer(compute="_compute_dedication")
+
+    @api.depends("task_ids")
+    def _compute_execution(self):
```

Ruff (0.8.2)

Missing return type annotation for private function `_compute_execution`

Add return type annotation: `None`

(ANN202)